azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/AbstractWebAppMojo.java

```diff
@@ -235,15 +235,15 @@
      * Location of the jar file which is going to be deployed. If this field is not defined,
      * plugin will find the jar file with the final name in the build directory.
      *
-     * @since 1.2.0
+     * @since 1.3.0
      */
     @Parameter(property = "webapp.jarFile")
     protected String jarFile;
 
      /**
-     * Jar command arguments to be used by web.config
+     * Jar command arguments to be used by web.config (Web App on Windows)

```

Comment:
We need to work on this feature and make this OS agnostic. <jarCommand> should be supported for JAR deployment, whether that's on Linux or Windows.

Reply:
Yes but currently the executable jar support for Web App is different between Windows and Linux at server side.

web.config is a hack for Windows Web App. Linux Web App do not have such kind of file.